BrainPortal/app/views/tools/_tools_table.html.erb

```diff
@@ -113,6 +113,9 @@
 
     t.column("Help", :help) do |tool|
       path = tool.cbrain_task_class.constantize.public_path("edit_params_help.html") rescue nil
+      if path.nil? # check if it is Boutiques tool
+        path = tool.cbrain_task_class.constantize.get_boutiques_help_filepath rescue nil
```

Comment:
Would this break (method does not exist) if path is nil but the tool isn't a Boutiques tool?